cdap-app-fabric/src/main/java/co/cask/cdap/test/RuntimeStats.java

```diff
@@ -161,4 +164,21 @@ public boolean apply(Map.Entry<String, AtomicLong> input) {
 
   private RuntimeStats() {
   }
+
+  /**
+   * Return the metrics key which which has the given starting and ending. If no such key is found returns an empty
```

Comment:
Returns the metrics key having the given starting and ending parts. If no such key is found, returns an empty string.